api/resources_portal/models/frontend_notifications_config.py

```diff
@@ -0,0 +1,141 @@
+            "on the condition that you provide the following items: {required_info_plain_text}"
+        )
+    },
+    "MATERIAL_REQUEST_REQUESTER_IN_FULFILLMENT": {
+        "body": (
+            "[organization_name]() has accepted your request for"
+            " [material_name]()} and is working to fulfill your request."
+        )
+    },
+    "MATERIAL_REQUEST_REQUESTER_EXECUTED_MTA": {
+        "body": (
+            "[organization_name]() has uploaded the fully executed MTA for your request for"
+            " [material_name]()} and is working to fulfill your request."
+        ),
+    },
+    "MATERIAL_REQUEST_REQUESTER_FULFILLED": {
+        "body": (
+            "[organization_name]() marked your request for"
+            " [material_name]()} as fulfilled.\nPlease view fulfilment notes for details."
+        )
+    },
+    "MATERIAL_REQUEST_REQUESTER_REJECTED": {
+        "body": (
+            "[organization_name]() rejected your request for [material_name]()} "
+            "for the below reason:<br>{rejection_reason}"
+        )
+    },
+    "MATERIAL_REQUEST_REQUESTER_CANCELLED": {
+        "body": "You cancelled your request for [material_name]()}.",
+    },
+    "MATERIAL_REQUEST_REQUESTER_ESCALATED": {
+        "body": (
+            "Request for {material_category}, [material_name]()} has been"
+            " escalated to the grants team.<br>"
+            "ALSF Grants team is looking into this and will be in touch with you soon.<br>"
+            "Below is the message they received from you for your records.\n{message}"
+        )
+    },
+    "MATERIAL_ADDED": {
+        "body": ("{other_name} added a new [material_name]()} to [organization_name]()."),
+    },
+    "MATERIAL_ARCHIVED": {
+        "body": ("{other_name} archived [material_name]()} from [organization_name]().")
+    },
+    "MATERIAL_DELETED": {
+        "body": ("{other_name} deleted [material_name]()} from [organization_name]()."),
+    },
+    "ORGANIZATION_NEW_MEMBER": {"body": "{other_name} was added to [organization_name]().",},
+    "ORGANIZATION_BECAME_OWNER": {
+        "body": (
+            "{other_name} has made you the owner of [organization_name]()."
+            "\nYou can now add new team members and remove members and resources."
+        ),
+    },
+    "ORGANIZATION_NEW_OWNER": {"body": "{other_name} is now the owner of [organization_name]().",},
+    "ORGANIZATION_MEMBER_LEFT": {"body": "{other_name} left [organization_name]().",},
+    "ORGANIZATION_NEW_GRANT": {
+        "body": (
+            "{other_name} linked a new grant {grant_name} with [organization_name]()."
+            "\nTeam members can now add resources associated with the grant."
+        )
+    },
+    "ORGANIZATION_INVITE": {
+        "body": "{organization_owner} has added you to their team, [organization_name]().",
+    },
+    "REPORT_TO_GRANTS_TEAM": {
+        "body": (
+            "Your issue has been reported to the ALSF Grants Team.<br>"
```

Comment:
Some of these are using `<br>` and some are using `\n`. It should probably be consistent right? (I wonder if that was also an issue in the old config as well)

Reply:
You're right. For this purpose, I changed it to all \n because we don't interpret this as HTML.